templates/admin/index_m.html.ts

```diff
@@ -91,19 +93,14 @@ ${useReact ? "" : (`
 			callback(obj);
 		}
 	</script>
-`)}
+`}
 </head>
 
 <body>
 
 ${useReact ? (`
 	<!-- this is where the React components are loaded into -->
-	<div class="m adapter-container">
-		<div class="loading">
-			<img src="${answers.adapterName}.png" />
-		</div>
-		<div class="translate loading">loading...</div>
-	</div>
+	<div class="m adapter-container" id="root"></div>
```

Comment:
This gets replaced with the rotating ioBroker logo, correct?

Reply:
It gets replaced with whatever is in App. And in the generated application, it will render the "Loading" animation defined in GenericApp.